inc/Engine/Admin/RocketInsights/AJAX/Controller.php

```diff
@@ -128,13 +129,40 @@ public function add_new_page(): void {
 		} else {
 			$page_title = $this->get_page_title( $payload['message'] );
 		}
-		$row_id = $this->manager->add_to_the_queue(
-			$url,
-			true,
-			[
-				'title' => $page_title,
-			]
-			);
+		$additional_details = [
+			'title' => $page_title,
+		];
+
+		// Try synchronous submission first.
+		$sync_response = $this->manager->attempt_sync_submission( $url, true, $additional_details );
```

Comment:
What do you think about using the `send_api` from `JobProcessor` [here](https://github.com/wp-media/wp-rocket/blob/df0e58c533575d5aa10688af989684fac62bdcd7/inc/Engine/Common/JobManager/JobProcessor.php#L334) instead re-inventing the wheel.